### ISSUE TITLE
Fill out the application

### ISSUE THREAD

**Issue by kurohune538** [GowanusGreenAR/new-view#5]
https://awrd.com/award/newviewawards2021/form

- Title

ambient divergence

- Detailed description of the work *(Not for public viewing)
Please tell us about the concept of your work and what you put into it. (Up to 400 characters) Please include any tools you used in the "description" above.

The pandemic forced a great withdrawal for many of us within our homes, computers, and interior worlds.  While this collective shift in behavior and worldview brought plenty of fear, suffering, and confusion, it also inspired an exponential rise in curiosity about new ways of communicating within these limitations. Recent transformations in xr technological affordances and cultural priorities have sparked massive industry-backed investment in the building of the "metaverse," or a more lifelike digital world. 

However, visions of this future vary greatly among the potential creators of it. Our project offers a dive into a particular vision of the metaverse that prioritizes nonhierarchical human connection and shared experience. You enter the world as a universal soul that can inhabit diverse individuals' points of view in scenes from their daily lives. As you explore more of the world(s), you add more energy to the universe's core, until the lightfield fills up, and the community comes together as a connected whole.

To create this work, we collaborated on iterations of scenes in the Unity game engine with the Styly plugin, Shuriken particle system, Playmaker plugin, Mixamo animations, Sketchfab 3D model platform, Github version control, and microphones from friends around the world (for recording ambient audio).

- URL of your work *(Not open to the public)
The URL of your work is the URL that will be issued when you click "Publish" on STYLY. URL example: http://gallery.styly.cc/colliu/69e9da7a-30d4-11e8-b34d-4783bb2170d0

- Creator name
hook.link

- Applicant's profile and biography *(Not open to the public) (up to 400 words)

hook.link is a design studio consisting of Jack Chen, Shin Komiya, and Lucas Wozniak, three friends who met at NYU's Interactive Telecommunications Program in 2020. They collaborate at the intersection of art, design, and emerging technology, with a focus on immersive and experiential installations. Themes that drive their works include intersubjectivity, perception, speculative futures, embodiment, and belonging.

- URL of a video introducing your work (within 3 minutes) *(Not for public viewing)
-

**Comment by ukpfilms** [comment]
Jack Bio:

Jack is a media artist with a passion for exploring the relationship between technology and society through visual storytelling, often through the lens of science fiction. Expanding on his background in traditional filmmaking, Jack is currently exploring new possibilities of storytelling through technology with medium such as virtual production, interactive VR experiences, and augmented reality.

**Comment by kurohune538** [comment]
Shinnosuke Komiya is a creative technologist and artist born in Tokyo and based in Brooklyn, NY. His research focuses on human augmentation, not only in terms of physicality but also by expanding one's imagination via his artworks. His works consist of multimedia explorations of interactions between physical and virtual phenomena. 
One of his representative works "GROOVE" was exhibited at SXSW in 2017. 
https://cham538.com

**Comment by sapphire-cision** [comment]
Lucas bio:

Lucas Wozniak is a multidisciplinary artist and researcher. His works try to make sense of interconnected social and ecological issues through multimedia experiences. His curiosity about how to make media environments more interactive, embodied, and personally meaningful for audiences has led to his growing interest in experience design, virtual and augmented reality development, and ai-assisted storytelling and worldbuilding.

**Comment by kurohune538** [comment]
By experiencing the other through first-person possession of other people in this world, it provokes that whether there is a normal.

**Comment by ukpfilms** [comment]
While technology has made information more accessible than ever, it also completely disrupted the notion of universal truths. Technologies invented with the vision of connecting communities often end up creating echo chambers amongst people, causing further misunderstanding the divisions as people drowns in a clashing sea of truths and misinformation. How can we re-orientate ourselves in this confusing world we've created, a world with no unified belief? (... then talk about how our project let's people see in other's shoes/perspective so people can understand eachother etc...)

**Comment by sapphire-cision** [comment]
The pandemic forced a great withdrawal for many of us within our homes, computers, and interior worlds. This collective shift in behavior and worldview brought plenty of fear, suffering, and confusion, but it also inspired an exponential rise in curiosity about new ways of communicating within these limitations.

While technology has made information and remote connection more accessible than ever, it also has disrupted the notion of universal truths and amplified rifts in cultural identity. Technologies invented with the vision of connecting communities often end up creating echo chambers amongst people, causing further misunderstanding and divisions as people lose their way in a clashing sea of truths and misinformation. 

How can we re-orient ourselves in this confusing world we've created, a world with no unified belief? Our project offers a glimpse of an answer to this question by allowing you to experience “radical otherness” through first-person possession of other people in our virtual world. It causes one to pause and reflect on innovative possibilities of cultivating more normative perceptions in abnormal times.

To create this work, we collaborated on iterations of scenes in the Unity game engine with the Styly plugin, Shuriken particle system, Playmaker plugin, Mixamo animations, Sketchfab 3D model platform, Github version control, and microphones from friends around the world (for recording ambient audio).
